Sources/CorePayments/Eligibility/SupportedPaymentMethods.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+enum SupportedPaymentMethodsType: String {
```

Comment:
I added this enum based on the logic that was previously removed: https://github.com/paypal/paypal-ios/blob/257df315bb8ab2e323df39bf17a5af48e36498a1/Sources/CorePayments/Networking/GraphQL/SupportedPaymentMethodsType.swift#L1